psydac/linalg/iterative_solvers.py

```diff
@@ -240,6 +241,7 @@ def pcg(A, b, pc, x0=None, tol=1e-6, maxiter=1000, verbose=False):
 def jacobi(A, b):
     """
     Jacobi preconditioner.
+    In case A is None we return a zero vector of the same dimensions as b
     ----------
     A : psydac.linalg.stencil.StencilMatrix | psydac.linalg.block.BlockMatrix
         Left-hand-side matrix A of linear system.
```

Comment:
Can you please add an empty line and a "Parameters" section?
```suggestion
    """
    Jacobi preconditioner.
    In case A is None we return a zero vector of the same dimensions as b.
    
    Parameters
    ----------
    A : psydac.linalg.stencil.StencilMatrix | psydac.linalg.block.BlockMatrix | None
        Left-hand-side matrix A of linear system.
```

Reply:
done in [abd53a7](https://github.com/pyccel/psydac/pull/213/commits/abd53a7daa92a80586489809d8c7b9c4578cd34b)